go/pkg/client/cas.go

```diff
@@ -1245,17 +1268,27 @@ func (c *Client) downloadBatch(ctx context.Context, batch []digest.Digest, reqs
 	LogContextInfof(ctx, log.Level(3), "Downloading batch of %d files", len(batch))
 	bchMap, err := c.BatchDownloadBlobs(ctx, batch)
 	if err != nil {
-		afterDownload(batch, reqs, err)
+		afterDownload(batch, reqs, map[digest.Digest]int64{}, err)
 		return
 	}
 	for _, dg := range batch {
 		data := bchMap[dg]
+		lbMoved := dg.Size
+		rbMoved := int64(len(data))
```

Comment:
This cannot ever be different from lbMoved here, right? This is BatchDownload, we don't compress these.